src/pages/components/feature-card-block.mdx

```diff
@@ -22,16 +22,39 @@ import ComponentFeedback from 'components/ComponentFeedback';
 ## Feature card block &mdash; medium
 Feature card block &mdash; medium is used to present a group of information with a medium size card with an image.
 
+### Content guidance
+
+| Field | Field type | Required   | Cardinality | Max character size (English / translated) |
+| -------- | --------- | ---------- | ---------- | ----------|
+| Headline | Short copy | Yes   | 1 | XS (65 / 85)|
```

Comment:
This leverages content group's headline. Just calling that out here.

Reply:
Yeah, I think I refrained from calling those out just cause it would be like for all components haha.